### DISCUSSION TITLE
Updating Map

### DISCUSSION THREAD

**Discussion by Jmccon** [bytedeck/bytedeck#1333] [General]
Hello, an FYI that I'm running into issues if wanting to update a Map. I get a 504 error with a link to this page.

**Comment by tylerecouture** [comment]
@Jmccon, what's your deck name? I'll take a look.

**Reply by tylerecouture** [reply]
> Hello, an FYI that I'm running into issues if wanting to update a Map. I get a 504 error with a link to this page.

I see the error message. Can you tell me what specifically you are doing to get that the error?

**Comment by Jmccon** [comment]
Hello, I made changes to a map setting, clicked a check box, and clicked update.
Jeff

Jeff McConkey (he/him/his)
Learning Specialist

***@***.***      ***@***.***
301 - 80 Station Street, Duncan, BC, V9L 1M4
Phone: 250-748-9880  |  Email: ***@***.***
workbccentre-duncan.ca<https://workbccentre-duncan.ca>  |  WorkBC Virtual<https://workbcvirtual.ethoscmg.com/>  |  workbc.ca<https://www.workbc.ca/>
[facebook-icon_64x64]<https://www.facebook.com/workbc.cowichan.valley/>  [twitter-icon_64x64] <https://twitter.com/workbc_cowichan>   [linkedin-icon_64x64] <https://www.linkedin.com/in/workbc-cowichan-valley>
ETHOS acknowledges with gratitude all the First Nations on whose traditional and unceded lands we work and provide service; in particular, the Snuneymuxw, Stz'uminus and Quw’utsun peoples, on whose lands our offices are located.
If you receive this e-mail in error, please notify the sender by reply e-mail and delete and destroy the message.
ü Please consider the environment before printing this email.

**Reply by tylerecouture** [reply]
@Jmccon If you can be more specific (which map were you editing, what changes were you trying to make, and what box did you tick) I can help you out!

**Comment by Jmccon** [comment]
Hi Tyler, I will try and reproduce. Not that I’m worried about it but thought to send along as an FYI. Is this a production environment? I was kind of testing what would happen, but ya not a good response from the system. Let me get back to you.
Thanks,

Jeff

Jeff McConkey (he/him/his)
Learning Specialist

***@***.***      ***@***.***
301 - 80 Station Street, Duncan, BC, V9L 1M4
Phone: 250-748-9880  |  Email: ***@***.***
workbccentre-duncan.ca<https://workbccentre-duncan.ca>  |  WorkBC Virtual<https://workbcvirtual.ethoscmg.com/>  |  workbc.ca<https://www.workbc.ca/>
[facebook-icon_64x64]<https://www.facebook.com/workbc.cowichan.valley/>  [twitter-icon_64x64] <https://twitter.com/workbc_cowichan>   [linkedin-icon_64x64] <https://www.linkedin.com/in/workbc-cowichan-valley>
ETHOS acknowledges with gratitude all the First Nations on whose traditional and unceded lands we work and provide service; in particular, the Snuneymuxw, Stz'uminus and Quw’utsun peoples, on whose lands our offices are located.
If you receive this e-mail in error, please notify the sender by reply e-mail and delete and destroy the message.
ü Please consider the environment before printing this email.

**Reply by tylerecouture** [reply]
Thank you! Playing around definitely shouldn't break anything and I would love to be ale to reproduce this bug so we can quash it.  Yes it's production environment (at least as production as it gets from a bunch of volunteers running a non-profit)
